package.json

```diff
@@ -2,24 +2,25 @@
   "name": "memize",
   "version": "1.1.0",
   "description": "Unabashedly-barebones memoization library with an aim toward speed",
-  "main": "index.js",
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "type": "module",
```

Comment:
This should make the package support usage as either a ES Module or CommonJS correct?